extensions/ql-vscode/src/common/mock-gh-api/request-handlers.ts

```diff
@@ -73,6 +73,7 @@ async function readRequestFiles(
     ) {
       request.response.body = await readFile(
         join(scenarioDirPath, request.response.body.substring(5)),
+        "utf8",
```

Comment:
I don't think this is equivalent/correct. All scenarios that have a `file:` response body have a ZIP file as their response and should not be decoded as UTF-8, but should be returned as-is.

Reply:
Ah sorry. I thought this was textual data. I can revert this bit and find another way around the type issues.